package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "4.0.0",
+	"version": "4.0.1-beta",
```

Comment:
This isnt a patch release, lets call this a Major and set it to `4.1.0-beta.1` and eventually `4.1.0`.

Reply:
I agree. Dev Console auth migration is currently published as a `4.1.0-alpha`. We'll need to update _**that**_ version to `5.0.0-alpha`